spec/spec_helper.rb

```diff
@@ -95,6 +95,7 @@ def describe_pattern(name, pattern_modes = [ nil ], &block)
       let(:pattern) { name }
       let(:message) { raise 'let(:message) { ... } is missing' }
       let(:grok) { grok_match(pattern, message) }
+      subject { grok }
```

Comment:
nit: named subjects can make the code that uses this easier to read.

In this case, we already are creating a memoized variable, so we could do so with `subject(name)` instead:

``` suggestion
      subject(:grok) { grok_match(pattern, message) }
```

OR, perhaps with a more descriptive name:

``` suggestion
      let(:grok) { grok_match(pattern, message) }
      subject(:grok_result) { grok }
```

Reply:
thanks put this on the wip branch: https://github.com/logstash-plugins/logstash-patterns-core/commit/f6e13cd8a83c5a62158128808b703bf643516bab